### ISSUE TITLE
Issue with Openset testset

### ISSUE THREAD

**Issue by delyan-boychev** [HighwayWu/LASTED#3]
I have found that some of the synthetic images in Artist&Danbooru folders reproduce real photos, but not paintings as mentioned in the paper. Moreover, there is no annotation file in the repository. How can I use the Openset test set and where can I find the annotation file?

**Comment by HighwayWu** [comment]
Thanks for your asking. 1) Do you mean that some "painting" is classified as "photo" in the Artist&Danbooru folders? 2) Since the Openset test is just for testing (e.g., a binary classification/identification problem), the annotation is only one of "real" and "fake" as the file name related.

**Comment by delyan-boychev** [comment]
1) While reading the paper I understood that Artist&Danbooru is a set of paintings, but some of the synthetic images in Artist&Danbooru folders in Openset are synthetic photos, but not synthetic paintings as mentioned. 
2) I'm trying to use it for multiclass classification - real photo, real painting, synthetic photo, synthetic painting.

**Comment by HighwayWu** [comment]
1. Yes. The images in Artist&Danbooru should be all paintings. Please specific the name of the wrong image, and I'll double check.
2. Sorry that the Openset is only for binary classification. Later the training dataset will be released, where multi-class evaluation could be conducted.

**Comment by HighwayWu** [comment]
Sorry for any confusion. In fact, the "Artist_Danbooru_StyleGAN/synthetic_1064_StyleGAN.png" means this is a synthetic image  generated by StyleGAN, rather than a real image (photo/painting) from "Artist_Danbooru". Only the images with "real_" prefix are from "Artist_Danbooru" (paintings) or "ImageNet_VISION" (photos).
